tests/e2e/utils/__fixtures__/price-benchmark-suggestions.json

```diff
@@ -0,0 +1,167 @@
+[
+	{
```

Comment:
The shape of these stubs don't match the expected shape being returned by the API. See: https://github.com/woocommerce/google-listings-and-ads/blob/a91842f08f41a88854ab9ef7549ae9c1256d5284/tests/Unit/API/Site/Controllers/MerchantCenter/PriceBenchmarksControllerTest.php#L96-L109

For this to integrate with the REST API, we need to align on an expected schema.

Reply:
Sounds good. Updated @joemcgill . I thought we were going to have a different format when we store all the data locally, but sounds good 😁

Reply:
Oh, I hadn't thought about that. Were you planning to change the shape in the reducer? I think keeping the shape the same will likely help with debugging since we can mock the REST API response using the same data that you're using in the E2E tests